swift/ql/lib/codeql/swift/dataflow/ExternalFlow.qll

```diff
@@ -497,9 +497,6 @@ predicate parseContent(AccessPathToken component, Content content) {
   or
   parseEnum(component, content)
   or
-  component.getName() = "ArrayElement" and
-  content instanceof Content::ArrayContent
-  or
```

Comment:
May I suggest we don't delete this case, but make it a synonym for `CollectionElement`.  We can carefully ensure we're not using it ourselves, but that way we'll avoid breaking any user code that uses `ArrayElement`.